apps/console/src/features/users/components/parent-user-invite-response-list.tsx

```diff
@@ -0,0 +1,314 @@
+/**
+ * Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { AlertTitle } from "@mui/material";
+import Alert from "@oxygen-ui/react/Alert";
+import Typography from "@oxygen-ui/react/Typography";
+import {
+    DataTable,
+    EmptyPlaceholder,
+    TableColumnInterface
+} from "@wso2is/react-components";
+import React, { ReactElement, ReactNode, useEffect, useState } from "react";
+import { Trans, useTranslation } from "react-i18next";
+import {
+    Grid,
+    Header,
+    Label
+} from "semantic-ui-react";
+import {
+    UIConstants, getEmptyPlaceholderIllustrations } from "../../core";
+import {
+    ParentOrgUserInvitationResult,
+    ParentOrgUserInviteErrorCode,
+    ParentOrgUserInviteResultStatus
+} from "../components/guests/models/invite";
+
+interface ParentInviteResponseListProps {
+    response: ParentOrgUserInvitationResult[];
+    isLoading?: boolean;
+    hasError?: boolean;
+    ["data-componentid"]?: string;
+}
+
+interface ResponseListItem {
+    username: string;
+    status: string;
+    message: string;
+}
+
+interface ResponseSummary {
+    successCount: number;
+    failedCount: number;
+    totalCount: number;
+}
```

Comment:
Shall we add these interfaces to models/invite.ts for better organization?

Reply:
These are this UI component specific interfaces, so I though instead of moving them to common invite model, keeping in the same file, WDYT?

Reply:
In that case, shall we use more descriptive variable names?

eg: 
ResponseSummary => ParentUserInviteSummaryResponse
ResponseListItem => ParentUserInviteResponseListItem

Reply:
Changed